addon/components/validated-input/template.hbs

```diff
@@ -31,7 +35,19 @@
         {{option.label}}</label>
     </div>
   {{/each}}
-
+{{else if (eq type "checkbox")}}
+  <div class="{{config.css.checkbox}}">
+    <label>
+      {{one-way-checkbox
+          checked
+          name       = name
+          value      = value
```

Comment:
What you describe I'd envision as a different component, which should probably be called `checkboxGroup`. In more general terms, I'd like to separate the following two cases for this addon:

1. Common use cases (Simple inputs, selects, checkboxes, radios, textareas, ... ): They should be supported out of the box with a very simple API (i.e. no need to specify the model) and should all feature proper validation support. 
2. Not-so-common use cases (Form elements with custom markup, special input elements): This addon shouldn't make it harder to include those compared to a plain `<form>`. Probably the simplest way to support this to create the changeset yourself in the controller and reference it directly in the custom component.

I think both the `checkbox` and `checkboxGroup` components fall in the first category, i.e. I'd like an API like this
```
{{f.input type="checkbox" label="I accept the terms and conditions" name="terms"}}
```
Where `model.terms` is just a `Boolean`; and
```
{{f.input type="checkboxGroup" label="My favorite kinds of chocolate" name="favorites" options=chocolates}}
```
with a model like this
```
chocolates: [{
  key: 'milk',
  label: 'Milk',
  value: true
}, {
  key: 'dark',
  label: 'Dark',
  value: false
}],
```
Where `label` could be optional (use `key` as fallback).

What do you think about this approach?